static/src/javascripts/projects/common/modules/audio/AudioPlayer.js

```diff
@@ -0,0 +1,641 @@
+// @flow
+/* eslint-disable no-nested-ternary */
+import {
+    React,
+    Component,
+    styled,
+} from '@guardian/dotcom-rendering/packages/guui';
+import palette, {
+    pillarsHighlight,
+} from '@guardian/dotcom-rendering/packages/pasteup/palette';
+import {
+    tablet,
+    leftCol,
+    wide,
+    mobile,
+    desktop,
+} from '@guardian/dotcom-rendering/packages/pasteup/breakpoints';
+import ophan from 'ophan/ng';
+
+import download from 'svgs/journalism/audio-player/download.svg';
+import pauseBtn from 'svgs/journalism/audio-player/pause-btn.svg';
+import playBtn from 'svgs/journalism/audio-player/play-btn.svg';
+import volume from 'svgs/journalism/audio-player/volume.svg';
+import fastBackward from 'svgs/journalism/audio-player/fast-backward.svg';
+import fastForward from 'svgs/journalism/audio-player/fast-forward.svg';
+import fastBackwardActive from 'svgs/journalism/audio-player/fast-backward-active.svg';
+import fastForwardActive from 'svgs/journalism/audio-player/fast-forward-active.svg';
+
+import { formatTime } from './utils';
+
+import ProgressBar from './ProgressBar';
+import Time from './Time';
+import config from '../../../../lib/config';
+
+const applePodcastImage =
+    config.get('images.journalism.apple-podcast-logo') || '';
+
+const AudioGrid = styled('div')({
+    borderTop: '1px solid #767676',
+    display: 'grid',
```

Comment:
Have you considered how this'll look for user's on browsers that don't support CSS `grid`? We still have a sizeable amount of users browsing using older browsers.

Reply:
@regiskuckaertz - as the creator of the player what is your response to this?
On my part I assumed our build process made the css backwards compatible

Reply:
yeah, we will care about that when we know we can move forward with that design of the player 👌

Reply:
How about limiting showing the experiment only to those with modern browsers?

Reply:
A good idea - however, because the A/B bucket splitting happens on the serverside in this case, the decision has already happened before we'd know about the browser. 
I will push to CODE and try and check on a few devices now

Reply:
I was assuming that we'd do it on the server, where you check `ActiveExperiments.isParticipating`. We can check the `User-Agent` header.

Reply:
Oh I see, I didn't think of that header. I think if I go back to amend code, I might as well add a layout fallback for the CSS because we will need to add that eventually. Test specific code is destined for the trash after next week.

Reply:
Great idea but would prefer this to be picked up by the dotcom team if useful. Parsing UA strings is something we shouldn't have to worry about.